### ISSUE TITLE
Document new AWS account limits for ECS

### ISSUE THREAD

**Issue by donstephan** [aws/copilot-cli#2685]
Hey all,

I just wanted to drop a note and recommend documenting the quotas and limits for new AWS accounts. I've dealt with this 3 times now with different AWS accounts that I've helped manage and deploy apps using Copilot. With new accounts you can easily get a error when rolling out a change for more than 2 Fargate launch types (since these new accounts apparently have different quota limits?). You need to go request a quota increase in order to get this all working well. It can be prohibitive for new developers looking to try out Copilot who don't have an already established AWS account.

```
Reason: You've reached the limit on the number of spot tasks you can run concurrently. For more information, see the Troubleshooting section of the Amazon ECS Developer Guide.
```

It's a frustrating error and will cause the Cloudformation script to spin continuously while failing deployment (haven't looked at why it doesn't hit a circuit breaker to eventually fail). It can be remedied by either manually cancelling the script update or updating the task in ECS and setting the task count to 0. Here's an image of the error happen when I'm simply trying to roll an update out on a staging server:

![Screenshot from 2021-08-03 10-20-06](https://user-images.githubusercontent.com/17347567/128031952-39328d1f-c486-4b82-9f68-7319283b89b6.png)

Do note I believe the limit is 2 tasks of each type but I cannot find any information in the AWS documentation which sheds light on this.

**Comment by Lou1415926** [comment]
Yes it does seem like new AWS accounts might start with lower limits than documented [here](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/service-quotas.html). This should indeed be documented better. Thank you for letting us know!

**Comment by joelbrandenburg** [comment]
We are publishing an update to the documentation to note that the service quota for concurrent Fargate tasks launched may vary and that new accounts may have a lower quota.  You should be able to view your current service quota in the Service Quotas console or via the Service Quotas API.  This is also an adjustable quota you can request an increase for.  The documentation updates should be live soon.

**Comment by Lou1415926** [comment]
I'm closing this issue now since the reported issue has been routed to the correct team and is being handled. Please feel free to reopen it or update with comments!

**Comment by MatiasMinian** [comment]
@joelbrandenburg Is this quota the one you are talking about?
<img width="1437" alt="Screen Shot 2021-10-13 at 21 02 39" src="https://user-images.githubusercontent.com/11709426/137196924-7fd0b20f-91d1-40db-ae66-5e18c91be0af.png">
My AWS account is pretty new but it seems I do have the quota. Unsure why I'm still getting this error 🤔

**Comment by efekarakus** [comment]
Hi @MatiasMinian !

Apologies for the late reply, you'll have the quota I believe but by default new AWS accounts will have some Fraud detection mechanism set in place.

I believe if you manually create a small EC2 instance, wait a few hours and then terminate it, it should get around this issue and you can spin up Fargate tasks. Apologies for the inconvenience 🙇

**Comment by awanishraj** [comment]
We started seeing the same error and upon checking Service Quotas, found that the applied quota was 2. It seems to have dropped recently as we have launched upto 20 tasks in the past. Even the current utilization value is 13 out of 2. Have raised a quota increase request, but not sure how long that will take. Any help on this?

<img width="977" alt="Screenshot 2022-04-12 at 1 13 43 PM" src="https://user-images.githubusercontent.com/6928709/162908506-632bbd9a-9a7f-414b-8e6d-3be588ee891c.png">
